cernopendata/modules/fixtures/data/docs/cms-guide-pileup-simulation/cms-guide-pileup-simulation.md

```diff
@@ -1,45 +1,105 @@
+2. The number of interactions per bunch crossing (in-time and out-of-time) in the event is randomly sampled from a poisson distribution with its mean equal to the valule chosen in (1). This determines how many pileup interactions are added to the simulated event. This value is recorded as the "number of pileup interactions" in the event.
 
-2. For each bunch crossing, both in- and out-of-time, the number of interactions is randomly sampled from a poisson distribution with a mean equal to the value chosen in (1). Note that this implies that all bunches have the same instantaneous luminosity, rather than allowing for bunch-to-bunch luminosity variation. In practice, the spread of bunch-by-bunch luminosities is much smaller than the poisson distribution, so this effect is negligible. One might evolve the simulation to include such features as the beginning or end of bunch trains, where the out-of-time pileup will have a significantly different luminosity structure (i.e., there is none, either before or after a given event). Since the bunch trains are quite long, this effect of ignoring these variations is likely to be negligible as well. The number of interactions that is actually included in each bunch crossing is also recorded. (`num_PU_vertices_`).
+Note that applying the same mean number of interactions to all the bunch crossings in an event imposes that all bunches have the same instantaneous luminosity. Neglecting bunch-to-bunch luminosity variations is a negligible effect: in practice, the spread of bunch-by-bunch luminosities is much smaller than the spread of the poisson distribution, and the long bunch trains used in the LHC make the effect of ignoring any variations from the bunch-train structure negligibly small as well.
 
-## Pileup Distributions from Data
+## Pileup distributions
 
-The pileup distributions measured from data are available in [the CMS Luminosity information page](https://twiki.cern.ch/twiki/bin/view/CMSPublic/LumiPublicResults).
+#### Measured from data
 
-## Monte Carlo Pileup Distributions
+The pileup distributions measured from data are available in the [CMS Luminosity information page](https://twiki.cern.ch/twiki/bin/view/CMSPublic/LumiPublicResults).
 
-The "ideal" distributions input for each year's Monte Carlo production samples are catalogued on [this page](https://twiki.cern.ch/twiki/bin/view/CMSPublic/Pileup_MC_Gen_Scenarios), which contains the actual numerical contents of each bin represented in the plots below.
+#### Used in Monte Carlo production
 
-### 2011
+The numerical contents of the estimated pileup distributions used in the MC productions can be found on [this Twiki page](https://twiki.cern.ch/twiki/bin/view/CMSPublic/Pileup_MC_Gen_Scenarios). The distribution varies for each year. The plots below are made from the numerical values (corresponding to each year's pileup scenario) provided on the TWiki page. The x-axis of the histograms below are "mean number of interactions per bunch crossing".
```

Comment:
Maybe remove "TWiki" from the text i.e. "this TWiki page" -> "this page" so that if the page is moved elsewhere we do not need to change the text, just the link
(x 3)